src/timedate/timedated.c

```diff
@@ -337,6 +480,70 @@ static int property_get_ntp_sync(
         return sd_bus_message_append(reply, "b", ntp_synced());
 }
 
+static int property_get_ntp_service_states(
```

Comment:
why have this new bus call? it looks awfully specific... The idea is that timedated abstracts clients from the details, but if we'd spill all this stuff to people, then this would contradict the purpose, no? I'd be very conservative with adding such a verbose new option